segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/ServiceStarterTest.java

```diff
@@ -15,35 +15,48 @@
  */
 package io.pravega.segmentstore.server.host;
 
+import io.pravega.segmentstore.server.host.health.ZKHealthContributor;
 import io.pravega.segmentstore.server.store.ServiceBuilderConfig;
 import io.pravega.segmentstore.server.store.ServiceConfig;
+import io.pravega.segmentstore.storage.impl.bookkeeper.ZooKeeperServiceRunner;
+import io.pravega.shared.health.Health;
+import io.pravega.shared.health.Status;
 import io.pravega.test.common.SerializedClassRunner;
-import io.pravega.test.common.TestingServerStarter;
-import lombok.Cleanup;
+import io.pravega.test.common.TestUtils;
 import org.apache.curator.framework.CuratorFramework;
-import org.apache.curator.test.TestingServer;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import java.io.IOException;
 
 @RunWith(SerializedClassRunner.class)
 public class ServiceStarterTest {
 
-    private String zkUrl;
-    private TestingServer zkTestServer;
+    private ServiceStarter serviceStarter;
+    private ZooKeeperServiceRunner zkService;
 
     @Before
-    public void startZookeeper() throws Exception {
-        zkTestServer = new TestingServerStarter().start();
-        zkUrl = zkTestServer.getConnectString();
+    public void setup() throws Exception {
+        int port = TestUtils.getAvailableListenPort();
+        zkService = new ZooKeeperServiceRunner(port, false, null, null, null);
```

Comment:
Why not using `TestingServerStarter` anymore?

Reply:
I can revert it, I was trying to match how it was done in the standalone for starting zkService.